aspnetcore/fundamentals/environments.md

```diff
@@ -87,16 +111,21 @@ These commands take effect only for the current window. When the window is close
 
 ![System Advanced Properties](environments/_static/systemsetting_environment.png)
 
-![ASPNET Core Environment Variable](environments/_static/windows_aspnetcore_environment.png) 
+Ha, Steve is setting a user env variable, not a system one. That won't do much good on a server.
+
+![ASPNET Core Environment Variable](environments/_static/windows_aspnetcore_environment.png)
+
+-->
 
 **web.config**
 
 See the *Setting environment variables* section of the [ASP.NET Core Module configuration reference](xref:hosting/aspnet-core-module#setting-environment-variables) topic.
 
 **Per IIS Application Pool**
 
-If you need to set environment variables for individual apps running in isolated Application Pools (supported on IIS 10.0+), see the *AppCmd.exe command* section of the [Environment Variables \<environmentVariables>](/iis/configuration/system.applicationHost/applicationPools/add/environmentVariables/#appcmdexe) topic in the IIS reference documentation.
+To set environment variables for individual apps running in isolated Application Pools (supported on IIS 10.0+), see the *AppCmd.exe command* section of the [Environment Variables \<environmentVariables>](/iis/configuration/system.applicationHost/applicationPools/add/environmentVariables/#appcmdexe) topic.
 
+<!--
```

Comment:
Why are some of these commented out?

Reply:
Why would you set the environment in a shell only to have that env variable gone when the shell exited? If that's useful, say for testing, we need to say that - and also say that it goes away with the shell. It seems to me folks on these OS's know how to do this or can easily find out. We shouldn't be in the business of documenting the OS.

If you'd like to keep them I fix them (the writing needs work). For example:

> Setting the current environment for macOS can be done in-line when running the application;

The `export` command sets environment variables on macOS:

Reply:
See my comment:  Ha, Steve is setting a user env variable, not a system one. That won't do much good on a server.

Better to avoid information not directly related to the article than to get it wrong.

Reply:
It is useful to set a specific environment before before running your app for testing. If it's not set then it will default to Production which disables most debugging features.

There's some ongoing work for dotnet run to honor launchsettings.